app/frontend/src/fileRemove.js

```diff
@@ -0,0 +1,147 @@
+window.GOVUK = window.GOVUK || {}
```

Comment:
This came from GDS?  I *think* it's some kind of modal library, but the mix of plain vanilla JS, JQuery, prototype definitions and business logic makes it very difficult to follow.  As it stands, I'm not happy to include this in our code.